src/main/java/pl/szczesniak/dominik/whattowatch/users/domain/UserService.java

```diff
@@ -19,8 +23,11 @@ public boolean exists(final UserId userId) {
 		return repository.exists(userId);
 	}
 
-	public UserId login(String username) {
-		return repository.findBy(username).get().getUserId();
+	public UserId login(final Username username, final UserPassword userPassword) {
+		return Optional.ofNullable(repository
+				.findBy(username.getValue())
```

Comment:
dodatkowo to formatowanie sugeruje jakbyśmy robili optionala z repository i na nim robili finda
powinno być 
repo.find()
.filter
.orElseTrhow